complyscribe/utils.py

```diff
@@ -88,7 +88,12 @@ def write_cac_yaml_ordered(file_path: pathlib.Path, data: Any) -> None:
     Serializes a Python object into a CaC content YAML stream, preserving the order.
     """
     yaml = YAML()
+    # align with CaC content yaml file style
     yaml.indent(mapping=4, sequence=6, offset=4)
+    yaml.explicit_start = True
+    # temp workaround to mitigate line length difference
+    # between CaC yamlfix and complyscribe ruamel.yaml
+    yaml.width = 110
```

Comment:
The update of `yaml.width = 110 ` helped more, but it can't resolve it completely. I retested the last scenario, and it still has one line longer than 110.

Reply:
Yes. I think we can improve this after this discussion https://github.com/ComplianceAsCode/content/discussions/13664 . Thanks for your test.